admin-dev/themes/new-theme/js/components/module-card.js

```diff
@@ -347,7 +347,7 @@ export default class ModuleCard {
           self.confirmPrestaTrust(result[moduleTechName]);
         }
 
-        $.growl.error({message: result[moduleTechName].msg});
```

Comment:
This one is aiming errors on install where status == false, but what about others growls ?

Should we put the same behavior on request error or whatever ? wdyt ? @Sinepel

Reply:
Maybe for the other `$.growl.error` but not need for the `$.grow.notice` ? wdyt ?

Reply:
I agree that errors needs to be read, while notice can be avoided, but maybe we can increase a bit the time for notice to make it easier to read !